exercise_day1/Ex1_15b_AmstromgPerfectPalindrome.java

```diff
@@ -0,0 +1,74 @@
+package exercise_day1;
+
```

Comment:
Print clear outputs instead of just true/false. Also handle another case where a number is none of these (prime, perfect, armstrong)